std_msgs/Header header
string model_name
HorseDetection[] detections
